### PR TITLE
Corp Mercs Style SL Pointer/Locator For The Third Fucking Time

### PR DESCRIPTION
**Pull request by Surrealaser**
:cl: Surrealistik
-Implements a responsive Corp Mercs style SL pointer/arrow that is highly functional unlike the near-worthless current one.
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Document the changes in the changelog.

**Comment by Rohesie** [comment]
The change in the helmet camera breaks the overwatch consoles. If you want that, do it in a different PR.

**Comment by Surrealaser** [comment]
@Rohesie Son of a bitch, I fucking forgot to replace the head with the ear reference and now it's going to result in a bunch of work splitting off the damn PR.

**Comment by Surrealaser** [comment]
@Rohesie Alright, according to @spookydonut , that should have removed the camera part which is now here: https://github.com/tgstation/TerraGov-Marine-Corps/pull/244

These are guaranteed to conflict with each other so they cannot both be test merged simultaneously.

**Comment by spookydonut** [comment]
Recommending this be merged.

**Comment by DominikPanic** [comment]
Still needs changelog documentation.

**Comment by Surrealaser** [comment]
> Still needs changelog documentation.

Done.

**Comment by DominikPanic** [comment]
This will need some profiling, also can you remove the "shield" when there's no SL? And possibly make the arrow a little smaller on the sides?

**Comment by DominikPanic** [comment]
Also, conflicts

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -219,67 +219,139 @@
...
 /obj/item/device/radio/headset/almayer/equipped(mob/living/carbon/human/user, slot)
-	if(slot == SLOT_EARS)
-		if(headset_hud_on)
-			var/datum/mob_hud/H = huds[MOB_HUD_SQUAD]
-			H.add_hud_to(user)
-			//squad leader locator is no longer invisible on our player HUD.
-			if(user.mind && user.assigned_squad && user.hud_used && user.hud_used.locate_leader)
-				user.hud_used.locate_leader.alpha = 255
-				user.hud_used.locate_leader.mouse_opacity = 1
+	if(slot == ITEM_SLOT_EARS)
```
Comment (Rohesie) [inline code/game/objects/items/devices/radio/headset.dm]: Do you understand the difference between `SLOT_EARS` and `ITEM_SLOT_EARS`?
One is a bitwise flag, the other is not.

Comment (Surrealaser) [reply code/game/objects/items/devices/radio/headset.dm]: copy pasta when spooky's conflict resolution didn't compile. It's fixed

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 2]
```diff
@@ -82,6 +82,35 @@
 
 	. += CONFIG_GET(number/outdated_movedelay/human_delay)
 
+/mob/living/carbon/human/Move()
+	. = ..()
+	Process_SL_Locator()
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/human/human_movement.dm]: StonedMC is in. This is not kosher and should be handled through a proper subsystem now.

**Comment by spookydonut** [comment]
Added 600 to the PR number #669

**Pull request closed without merging**